docs/operator-manual/core.md

```diff
@@ -0,0 +1,95 @@
+# Argo CD Core
+
+## Introduction
+
+Argo CD Core is a different installation that runs Argo CD in headless
+mode. With this installation, you will have a fully functional GitOps
+engine capable of getting the desired state from Git repositories and
+applying it in Kubernetes.
+
+The following groups of features won't be available in this
+installation:
+
+- Argo CD RBAC model
+- Argo CD API
+- OIDC based authentication
+- Argo CD web application
+- Argo CD CLI
```

Comment:
Is this completely true? I thought some CLI commands have flags which allow you to perform equivalent actions using the k8s API instead of the Argo CD API.

Reply:
Actually in the usage section below I am documenting how the CLI and UI can still be used. I created a new section for partially available features.